backend/import-and-run.sh

```diff
@@ -2,4 +2,13 @@
 
 keytool -import -trustcacerts -keystore /usr/lib/jvm/java-1.8-openjdk/jre/lib/security/cacerts -storepass changeit -noprompt -alias postgrescert -file /pg-certs/server.crt
 
-./run-as-user.sh lein repl :headless
+if [ -z "$1" ]; then
+    ./run-as-user.sh lein repl :headless
+elif [ "$1" == "functional-and-seed" ]; then
+    # Two thing in one so that we avoid starting yet another JVM
+    ./run-as-user.sh lein do test :functional, run -m dev/migrate-and-seed
+elif [ "$1" == "kubernetes-test" ]; then
+    ./run-as-user.sh lein test :kubernetes-test
```

Comment:
Do we have tests annotated with `:kubernetes-test`?

Reply:
No, I was planning to add one but now I realize that those will be with puppeteer.